examples/gazebo_scene_viewer/SceneManager.cc

```diff
@@ -1509,13 +1510,14 @@ void SubSceneManager::ProcessCone(
 void SubSceneManager::ProcessCapsule(
     const gazebo::msgs::Geometry & _geometryMsg, VisualPtr _parent)
 {
-  GeometryPtr capsule = this->activeScene->CreateCapsule();
-  const gazebo::msgs::CapsuleGeom &capsuleMsg = _geometryMsg.capsule();
-  double x = 2 * capsuleMsg.radius();
-  double y = 2 * capsuleMsg.radius();
-  double z = capsuleMsg.length();
-  _parent->SetLocalScale(x, y, z);
-  _parent->AddGeometry(capsule);
+  // \todo(anyone) needs gazebo capsule msg
```

Comment:
This package should use `ign-msgs7` and the message it's there https://github.com/ignitionrobotics/ign-msgs/blob/main/proto/ignition/msgs/geometry.proto

Reply:
the errors are actually related to `gazebo::msgs` (we have these msgs in ignition but not in gazebo). I think we would need to add the cone and capsule msgs in gazebo: https://github.com/osrf/gazebo/tree/gazebo11/gazebo/msgs

Reply:
I created an issue https://github.com/ignitionrobotics/ign-rendering/issues/293